graph/export.go

```diff
@@ -13,16 +13,15 @@ import (
 	"github.com/docker/docker/registry"
 )
 
-// CmdImageExport exports all images with the given tag. All versions
-// containing the same tag are exported. The resulting output is an
-// uncompressed tar ball.
-// name is the set of tags to export.
-// out is the writer where the images are written to.
+// ImageExportConfig holds list of names to be exported to a output stream.
 type ImageExportConfig struct {
-	Names     []string
+	// Names is the set of tags to export.
+	Names []string
+	// OutStream is the writer where the images are written to.
 	Outstream io.Writer
 }
 
+// ImageExport exports list of images to a output stream specified in the config.
```

Comment:
You could add here that it uses tar to write on the ``ImageExportConfig.Outstream`` maybe.